src/adapters/prices-v2/tests/snapshots/ethereum.prices.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blockNumber": 19283451,
+  "blockNumber": 19311615,
```

Comment:
Quite cool, this has been picked up by the new snapshot builder and now it has added the block number to the testcase as well. So it's working for both protocolTokenAddresses and blockNumber.